impala/tests/test_dbapi_connect.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2013 Cloudera Inc.
```

Comment:
These tests look great!  Though I need to see if I have access to internal test clusters for all the options.